### PR TITLE
[guardian app] setup of application project

### LINKED ISSUE #6: [guardian app] Setup application
This task pertains to scaffolding the desktop applications used to facilitate the ceremony election guardians participate in. Ideally, we would use https://tauri.app/ to implement this.

### PR DESCRIPTION
**Pull request by soerenbf**
## Purpose

Sets up the project for the guardian desktop application.

Closes #6 

## Changes

- Move styling shared between apps into separate project.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: apps/guardian/.gitignore [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+lerna-debug.log*
+
+node_modules
+dist
+dist-ssr
+*.local
+
+# Editor directories and files
+.vscode/*
+!.vscode/extensions.json
```
Comment (abizjak) [inline apps/guardian/.gitignore:17]: Can we please not have this in the repo?

File: apps/guardian/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        "@types/react": "^18.2.15",
+        "@types/react-dom": "^18.2.7",
+        "@vitejs/plugin-react": "^4.2.1",
+        "eslint": "^8.56.0",
+        "prettier": "^3.1.1",
+        "typescript": "^5.0.2",
+        "vite": "^5.0.0",
+        "vite-tsconfig-paths": "^4.2.3"
+    }
+}
```
Comment (abizjak) [inline apps/guardian/package.json:36]: This comment is in the wrong place, but I cannot put it on the image. But can you please remove tauri.svg icon from the repo? I suppose we should use the concordium logo instead.

Comment (abizjak) [reply apps/guardian/package.json:36]: Samve for  vite.svg.

Comment (abizjak) [reply apps/guardian/package.json:36]: Please also replace all the icons/*.png with Concordium ones.

Tauri comes with a tool to generate the right ones from an image https://tauri.app/v1/guides/features/icons/

File: apps/guardian/src-tauri/tauri.conf.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            {
+                "fullscreen": false,
+                "resizable": true,
+                "title": "Concordium Election Guardian",
+                "width": 800,
+                "height": 600
+            }
+        ]
+    }
+}
```
Comment (abizjak) [inline apps/guardian/src-tauri/tauri.conf.json:45]: Please remove react.svg from the repository.

File: apps/guardian/package.json (hunk 2 above)
Comment (soerenbf) [reply apps/guardian/package.json:36]: thanks for the link 😄

**Pull request merged**